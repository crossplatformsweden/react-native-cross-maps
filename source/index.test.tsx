import { MapComponent } from './index';

jest.unmock('react-native');

describe('components', () => {
  describe('<MapView />', () => {
    it('Component should render', () => {
      expect(MapComponent).toMatchSnapshot();
    });
  });
});
