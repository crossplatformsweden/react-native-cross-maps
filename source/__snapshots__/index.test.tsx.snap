// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <MapView /> Component should render 1`] = `[Function]`;
